### ISSUE TITLE
(Property Import) Error: Cannot redeclare add_log()

### ISSUE THREAD

**Issue by mattsims** [propertyhive/WP-Property-Hive#134]
Hi guys,

I've seen several instances of this error in the logs since updating to the latest version (1.2.10) of the Property Import plugin:
```
WordPress database error Column 'instance_id' cannot be null for query INSERT INTO `bea_ph_propertyimport_logs_instance_log` (`instance_id`, `severity`, `entry`, `log_date`) VALUES (NULL, '1', 'Error:Cannot redeclare add_log() (previously declared in /nas/content/live/bridgesestate/wp-content/plugins/propertyhive-property-import/download-queued-media-cron.php:283)|1|propertyhive-property-import/download-queued-media-cron.php|283|Array\\n(\\n [0] ', '2020-09-29 12:56:08') /* From [www.bridges.co.uk/wp-cron.php?doing_wp_cron] in [/nas/content/live/bridgesestate/wp-content/plugins/propertyhive-property-import/cron.php:27] */ made by fatal_handler
```
Let me know if you need any more info 👍 

Regards,

Matt.

**Comment by propertyhive** [comment]
Hi Matt, 

Thanks for raising this. We have seen this on a few sites now. It shouldn't have any impact on the imports running. 

This is on our roadmap (https://trello.com/c/WWDEZEvJ) to be addressed in the next few days.

I'll keep this issue open for now and comment/close it once it's been fixed and a new version is available.

Thanks,
Steve

**Comment by propertyhive** [comment]
To confirm this should now be fixed as per the latest version of the import add on just released (v1.2.12).

Many thanks,
Steve
